src/services/determineIsInfoPage.ts

```diff
@@ -0,0 +1,15 @@
+import { formElementsService } from '@oneblink/sdk-core'
+import { FormTypes } from '@oneblink/types'
+
+const infoPageElements: FormTypes.FormElementType[] = [
+  'heading',
+  'html',
+  'image',
+  'section',
+]
+export default function determineIsInfoPage(form: FormTypes.Form) {
+  formElementsService.forEachFormElement(form.elements, (e) => {
+    if (!infoPageElements.includes(e.type)) return false
```

Comment:
I dont think this will work. this return statement is in an function that is not the `determineIsInfoPage` function.

Using the `findFormElement()` element would be more preformant as it will stop when it finds an element that matches the predicate, instead of iterating over all of the elements.

Reply:
Pretty sure it will work, but your right about the second part.